src/components/projects/ProjectsAPI.js

```diff
@@ -43,3 +43,14 @@ export async function getProject(id) {
     return handlerError;
   }
 }
+
+export async function postProject(project) {
+  try {
+    const route = process.env.REACT_APP_API_HOST;
```

Comment:
this is broken in prod, right? @rodrigo-baldiviezo-jalafund

Reply:
Done